source/strings.tex

```diff
@@ -2623,6 +2614,16 @@
 \pnum
 \returns
 \tcode{*this}.
+
+\pnum
+\throws
+\begin{itemize}
+\item \tcode{out_of_range} if \tcode{pos1 > size()},
+\item \tcode{length_error} if the length of the resulting string
+would exceed \tcode{max_size()} (see below), or
```

Comment:
`max_size()` is defined above, so maybe the "see below" refers to the now above _Effects:_ for when the length would exceed `max_size()`, which should be redundant given [string.require]p1.
```suggestion
would exceed \tcode{max_size()} (see above), or
```
```suggestion
would exceed \tcode{max_size()}, or
```

Reply:
Fixed (second option)